### PR TITLE
[Test] unittests for Breadcrumb.class.inc

### PR DESCRIPTION
**Pull request by AlexandraLivadas**
### Brief summary of changes
I made a new unit test class for Breadcrumb.class.inc
I might add more edge cases (like whitespace) if necessary.

### This is related to: 
https://github.com/aces/Loris/pull/4378

### To test this change...
Travis

### CODE REVIEW

File: test/unittests/BreadcrumbTest.php [hunk 1]
```diff
@@ -0,0 +1,78 @@
...
+     /**
+     * This method is called before each test is executed.
+     *
+     * @return void
+     */
+    protected function setUp(): void
+    {
+        parent::setUp();
+
+	$this->label = "testLabel";
```
Comment (kongtiaowang) [inline test/unittests/BreadcrumbTest.php]: need an indent

Comment (kongtiaowang) [reply test/unittests/BreadcrumbTest.php]: need to indent line:58-59

**Comment by christinerogers** [comment]
@AlexandraLivadas  This is great!  
If there is a To Do list left over (like whitespace) -- add it in the PR description header and/or (preferably) in a new Issue.
